sympy/functions/elementary/tests/test_piecewise.py

```diff
@@ -1553,3 +1553,22 @@ def test_issue_22533():
 def test_issue_24072():
     assert Piecewise((1, x > 1), (2, x <= 1), (3, x <= 1)
         ) == Piecewise((1, x > 1), (2, True))
+
+def test_issue_24127():
+    x = symbols('x', real=True)
+    f1 = Piecewise((sin(x), x<0), (S(1)/2, True))
+    x0 = 1
+    limit_test = f1.limit(x,x0)
+    assert limit_test == S.Half
+
```

Comment:
`Piecewise._eval_is_meromorphic` looks good but it will solve only a part of #24127. Hence I think that this test should be removed. The next test should suffice.

Reply:
Done.